onnxruntime/core/session/inference_session.h

```diff
@@ -512,6 +512,11 @@ class InferenceSession {
   const long long kDurationBetweenSending = 1000 * 1000 * 60 * 10;  // duration in (us).  send a report every 10 mins
   std::string event_name_;                                          // where the model is loaded from: ["model_loading_uri", "model_loading_proto", "model_loading_istream"]
 
+  TimePoint time_sent_last_evalutation_start_;
```

Comment:
Would you mind grouping all the telemetry related members in a struct of it's own? It would make the class easier to read and maintain. This would be a good time to do it.

Reply:
grouped into a struct